docs/advanced/advanced.md

```diff
@@ -316,12 +317,34 @@ It is recommended to turn off firewalld:
 systemctl disable firewalld --now
 ```
 
+If you wish to keep firewalld enabled, the following rules are required:
+```bash
+firewall-cmd --permanent --add-port=6443/tcp # flannel
```

Comment:
this is the supervisor/apiserver port. flannel also needs to be opened, as do the etcd ports if that's in use. Maybe cross-link to https://docs.k3s.io/installation/requirements#networking ?